packages/dapp/src/utils/crypto.ts

```diff
@@ -0,0 +1,51 @@
+const forge = require('node-forge');
```

Comment:
Not sure that the "node-forge" is the right choice as a cryptographic library instead of the initially proposed "crypto-js".

Just compare the amount of code required from the CLI project https://github.com/windingtree/org.id-cli/blob/c41c15e90ed5a2c15883b904605ebe822aada2b9/src/api/common.ts#L88

approx 10 (flattened) lines of code opposite to yours 47 with a lot of configurations.

I think - less code is better. Also, in the future, we can move encryption utilities to the shared library and use them across all of our projects. So, it is better to use the simplest implementation.